src/js/settings/settings.js

```diff
@@ -165,7 +165,7 @@ Settings.onOpenConfig = function(e) {
     return;
   }
   var hash = encodeURIComponent(JSON.stringify(options));
-  Pebble.openURL(url + '#' + hash);
+  Pebble.openURL(url + (listener.params.noHash ? '' : '#' + hash));
```

Comment:
Thanks for catching this! How about `hash: false` similar to ajax's `async: false`?
```js
if (listener.params.hash !== false) {
  url += '#' + encodeURIComponent(JSON.stringify(options));
}
Pebble.openUrl(url);
```
I thought about `options: false` too, but with that name, it should also disable option parsing. `options: 'save'` or `'saveOnly'` may be too verbose too.